### PR TITLE
Manual mutations

### PR DESCRIPTION
**Pull request by robinandeer**
Specify for each job if it requires manual intervention
Needed for automation

### CODE REVIEW

File: moneypenny/flow/actions/familydir.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from moneypenny.flow import types
 
 
-def dispatch(db, mutation_type, case_id):
+def dispatch(db, mutation_type, case_id, manual=False):
     """Dispatch mutation that only relies on family_dir property."""
```
Comment (ingkebil) [inline moneypenny/flow/actions/familydir.py]: Don't forget to expand the documentation

File: moneypenny/flow/queue.py [hunk 2]
```diff
@@ -50,13 +51,17 @@ def requeue(self, job_obj):
...
-            self.commit()
-        return job_obj
+    def pop(self, manual=True):
+        """Pop a job from the queue.
+
+        Args:
+            manual (bool): include jobs that need manual intervention
+        """
+        for job_obj in Job.query:
+            if manual or not job_obj.is_manual:
```
Comment (ingkebil) [inline moneypenny/flow/queue.py]: Just curious: how comes the `job_obj.is_manual` is negated here?

Comment (robinandeer) [inline moneypenny/flow/queue.py]: I want to say; if you want to run manual jobs or the job doesn't require manual intervention

Make sense?

Comment (ingkebil) [inline moneypenny/flow/queue.py]: Aha yes. I read it as "if the job is run manual and is not run manual", which didn't make sense.

**Pull request merged**